frontend/src/scenes/pipeline/hogfunctions/HogFunctionInputs.tsx

```diff
@@ -201,15 +202,15 @@ function DictionaryField({ onChange, value }: { onChange?: (value: any) => void;
                         placeholder="Key"
                     />
 
-                    <LemonInput
-                        className="flex-1"
+                    <CodeEditorInline
```

Comment:
One other small thing.

Would be cool if we could make it even more subtle and get rid of the right click behaviour. And maybe have a less "code-y" font (but thats not so important)
<img width="347" alt="Screenshot 2024-06-26 at 09 21 27" src="https://github.com/PostHog/posthog/assets/2536520/b21c3c36-c7cb-43ac-9f2f-fa90e25f3d67">

Reply:
Will address in a follow up. Can't waste the green state of this PR... 😅